newdle_frontend.js

```diff
@@ -29,18 +30,29 @@ fetch('http://localhost:5000/Game_info')
 .then(json_response => {
     getHeadlines(json_response)
 });
+// test feature, ***DELETE*** 
```

Comment:
What is this for?

Reply:
I was originally going to hard code in the previous headline as well, because otherwise when the API updates it's no longer possible to test the test headline (which has only one word scrambled). I will remove this